FutureCollars/src/main/java/lesson5task1/Text.java

```diff
@@ -0,0 +1,31 @@
+package lesson5task1;
+
+import java.util.Scanner;
+
+public class Text {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Hello world");
+        String text = scanner.nextLine();
+
+        if (text.length() == 0) {
```

Comment:
Wydziel proszę metodę w stylu isTextEmpty, jest to dobra praktyka zgodna z clean code, aby dzielić kod na mniejsze metody :)